week-1/exercises/random.js

```diff
@@ -7,3 +7,6 @@ const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 // Try breaking down the expression and using documentation to explain what it means
 // It will help to think about the order in which expressions are evaluated
 // Try logging the value of num several times to build an idea of what the program is doing
+ const minImum = Math.floor(sum)
+ let.Math.floor(Math.random() * 6 + 1);
```

Comment:
I don't understand whats line 11 doing, can you explain what you are you trying to achieve here please?